src/notation/internal/notationactioncontroller.cpp

```diff
@@ -578,7 +578,7 @@ void NotationActionController::moveAction(const actions::ActionCode& actionCode)
         return;
     }
 
-    Element* element = interaction->selection()->element();
+    Element* element = interaction->selection()->elements().back();
```

Comment:
I think, needs to check on `elements` not empty

Reply:
Fixed!